README.md

```diff
@@ -79,15 +79,14 @@ make run
 
 Alternatively, you could also run the application with
 [podman-compose](https://github.com/containers/podman-compose) by setting the
-`CACHITO_COMPOSE_ENGINE` variable (for now SELinux must be set to the
-**Permissive** mode before running the make command):
-
-> :warning: **Disabling SELinux or running it in Permissive mode may be
-> dangerous. Do it at your own risk and make sure you re-enable it after
-> running your integration tests.**
+`CACHITO_COMPOSE_ENGINE` variable to the path of the `podman-compose` script.
+Unfortunately, the latest release of `podman-compose` contains various bugs making
+it unusable for running Cachito locally. Use the script from the `devel` branch instead.
+To facilitate this, set `CACHITO_COMPOSE_ENGINE` to the special value `podman-compose-auto`.
+which will instruct the Makefile to download and use the correct version of `podman-compose`.
```

Comment:
It might also be worth mentioning that the script is downloaded into ./tmp, so after `make run` executes, you can interact with the deployment via `tmp/podman_compose.py ...` (or your pre-installed `podman-compose`, which seems to work fine for most things when stuff is already running)

(I was looking for it in /tmp at first...)

Reply:
Added to the docs.